README.md

```diff
@@ -11,10 +11,10 @@
 - Bootstap
 
 ## Video Describing the project features
```

Comment:
Thank you for your presentation. Kindly record a new video which only highlights a piece of code which you are proud of and find interesting.

Please add integretion specs to test if all the parts of the app work together without any issues.

Please use the fonts and colors already defined in the design page for the whole project. The `Add Category` button should have `green` color and it should be fixed on the bottom of the page.

Reply:
Hi @SajjadAhmad14, the add category has already green color :)